### PR TITLE
Refactor/tests

### PR DESCRIPTION
**Pull request by orsagie**
- [x] Ready for review
- [x] Follows [CONTRIBUTING](https://github.com/snyk/snyk/blob/master/.github/CONTRIBUTING.md) rules
- [ ] Reviewed by Snyk internal team

#### What does this PR do?
Refactor of tests

### CODE REVIEW

**Comment by dkontorovskyy** [comment]
Maybe move `cli-test.${ecosystem}.test.ts` to `cli-test` folder and have `${ecosystem}.test.ts`? Probably would be more readable.

File: test/acceptance/cli-monitor/cli-monitor.acceptance.test.ts [hunk 1]
```diff
@@ -1373,154 +1351,6 @@ test('`wizard` for unsupported package managers', async (t) => {
   });
 });
 
-test('`protect` for unsupported package managers', async (t) => {
```
Comment (orsagie) [inline test/acceptance/cli-monitor/cli-monitor.acceptance.test.ts]: These tests are already in `cli-protect.test.ts`

File: test/acceptance/cli-monitor/cli-monitor.acceptance.test.ts [hunk 2]
```diff
@@ -949,29 +950,6 @@ test('`monitor golang-app --file=vendor/vendor.json`', async (t) => {
   );
 });
 
-test('`test cocoapods-app (autodetect)`', async (t) => {
```
Comment (orsagie) [inline test/acceptance/cli-monitor/cli-monitor.acceptance.test.ts]: moved to `cli-test`

File: test/acceptance/cli-monitor/cli-monitor.acceptance.test.ts [hunk 3]
```diff
@@ -1535,51 +1365,6 @@ function stubExec(t, execOutputFile) {
   });
 }
 
-test('error 401 handling', async (t) => {
```
Comment (orsagie) [inline test/acceptance/cli-monitor/cli-monitor.acceptance.test.ts]: These are already in `cli-test` tests

File: test/acceptance/cli-monitor/cli-monitor.acceptance.test.ts [hunk 4]
```diff
@@ -3,10 +3,10 @@ import * as path from 'path';
 import * as fs from 'fs';
 import * as sinon from 'sinon';
 import * as needle from 'needle';
-import * as cli from '../../src/cli/commands';
-import { fakeServer } from './fake-server';
-import * as subProcess from '../../src/lib/sub-process';
-import * as version from '../../src/lib/version';
+import * as cli from '../../../src/cli/commands/index';
```
Comment (dkontorovskyy) [inline test/acceptance/cli-monitor/cli-monitor.acceptance.test.ts]: It's a nit peak, nothing more. Cos import will by default import `index` file. You can remove all the `index` occurrences

File: test/acceptance/cli-test/cli-test.acceptance.test.ts [hunk 5]
```diff
@@ -0,0 +1,154 @@
+import * as tap from 'tap';
+import * as cli from '../../../src/cli/commands/index';
```
Comment (dkontorovskyy) [inline test/acceptance/cli-test/cli-test.acceptance.test.ts]: It's a nit peak, nothing more. Cos import will by default import `index` file. You can remove all the `index` occurrences

File: test/acceptance/cli-test/cli-test.acceptance.test.ts [hunk 6]
```diff
@@ -0,0 +1,154 @@
...
+const apiKey = '123456789';
+let oldkey;
+let oldendpoint;
+let versionNumber;
+const server = fakeServer(process.env.SNYK_API, apiKey);
+const before = tap.runOnly ? only : test;
+const after = tap.runOnly ? only : test;
+
+// Should be after `process.env` setup.
+import * as plugins from '../../../src/lib/plugins/index';
```
Comment (dkontorovskyy) [inline test/acceptance/cli-test/cli-test.acceptance.test.ts]: It's a nit peak, nothing more. Cos import will by default import `index` file. You can remove all the `index` occurrences

**Pull request merged**